src/wallet/gtest/test_orchard_wallet.cpp

```diff
@@ -110,7 +110,7 @@ TEST(TransactionBuilder, OrchardToOrchard) {
 
     // If we attempt to get spend info now, it will fail because the note hasn't
     // been witnessed in the Orchard commitment tree.
-    EXPECT_THROW(wallet.GetSpendInfo(notes, wallet.GetLatestAnchor()), std::logic_error);
+    EXPECT_THROW(wallet.GetSpendInfo(notes, 1, wallet.GetLatestAnchor()), std::logic_error);
```

Comment:
Why is this 1 rather than 0?

Reply:
Oh I see, `OrchardWallet::GetSpendInfo` uses `anchorConfirmations - 1`.

Hmm, it that seems potentially error prone to use different conventions at the different layers, though.